packages/next/client/index.tsx

```diff
@@ -466,6 +466,14 @@ export function renderError(renderErrorProps: RenderErrorProps): Promise<any> {
   console.error(err)
   return pageLoader
     .loadPage('/_error')
+    .then(({ page: ErrorComponent, styleSheets }) => {
+      return lastAppProps?.Component === ErrorComponent
+        ? import('../pages/_error').then((m) => ({
```

Comment:
Seems that `import()` is compiled to `require()` and it's being bundled into the main.js bundle while that should not happen 🤔

Reply:
@timneutkens the code split will only happen when size > 20kb here. should I continue with current situation that `_error` is bundled into main chunk?